dashboard/app/controllers/programming_environments_controller.rb

```diff
@@ -4,7 +4,7 @@ class ProgrammingEnvironmentsController < ApplicationController
 
   before_action :require_levelbuilder_mode_or_test_env, except: [:index, :show, :docs_show, :docs_index]
```

Comment:
I have a follow-up item here to make this less of a sharp corner but you'll want to add `get_summary_by_name` to this list. If you don't, the route won't be accessible on production.